### PR TITLE
Shraddhavp-Blog-post

### PR DESCRIPTION
**Pull request by shraddhavp**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by Rohitesh-Kumar-Jain** [comment]
Hi, @shraddhavp 

Please make sure the CI turns green by fixing all the issues showed by the GitHub Actions : )

You may try running tests locally on your pc before pushing your changes on Github. 

To run tests locally use ```npm test```

**Comment by shraddhavp** [comment]
> Hi, @shraddhavp
> 
> Please make sure the CI turns green by fixing all the issues showed by the GitHub Actions : )
> 
> You may try running tests locally on your pc before pushing your changes on Github.
> 
> To run tests locally use `npm test`

Yes tried still it shows the same @Rohitesh-Kumar-Jain

**Comment by Rohitesh-Kumar-Jain** [comment]
> > Hi, @shraddhavp
> > Please make sure the CI turns green by fixing all the issues showed by the GitHub Actions : )
> > You may try running tests locally on your pc before pushing your changes on Github.
> > To run tests locally use `npm test`
> 
> Yes tried still it shows the same @Rohitesh-Kumar-Jain

You need to fix issues pointed by tests, for e.g. one of the issues pointed in your test 

```MD018/no-missing-space-atx No space after hash on atx style heading [Context: "##Project summary"]```

might be simply fixed by editing ```##Project summary``` to ```## Project summary```

you may fix the remaining issues in a similar manner : )

File: jekyll/_posts/2021-05-05- Shraddhavp-interactive-wechaty-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Shraddha  Vasant Prasad: 2021 Google Season of Docs Technical Proposal"
```
Comment (huan) [inline jekyll/_posts/2021-05-05- Shraddhavp-interactive-wechaty-tutorial.md]:
```diff
- Shraddha  Vasant
+ Shraddha Vasant
```

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for submitting your great post!

Please rename the file name to not contains spaces and all to be lowercase so that it can make the CI happier.

```diff
- jekyll/_posts/2021-05-05- Shraddhavp-interactive-wechaty-tutorial.md
+ jekyll/_posts/2021-05-05-shraddhavp-interactive-wechaty-tutorial.md
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shraddhavp Change folder name`05-Gsod-2021-techwriter-blogpost` to `05-shraddhavp-interactive-wechaty-tutorial`.

Format the markdown file something like this

```markdown
---
title: "Shraddha Vasant Prasad: 2021 Google Season of Docs Technical Proposal"
author: shraddhavp
categories: proposal
tags:
  - Documentation
  - gsod
  - 2021
  - project
  - proposal
image: /assets/2021/05-shraddhavp-interactive-wechaty-tutorial/ShraddhaPic.jpg
---

I am Shraddha Vasant Prasad, 2020 graduate and currntly working as a Software Engineer At American Express, I describe myself as "Developer in the making and a Blogger by passion"

## Technical writer Details

- Name: Shraddha Vasant Prasad
- Github: <https://github.com/shraddhavp>
- Linkedin: <https://www.linkedin.com/in/shraddha-v-prasad/>
- Blog: <https://medium.com/@anitha.shraddha>
- Location: Bangalore,Karnatak,India
- Time Zone: (UTC + 05:30)

## Project name

Create easy to use tutorial for Wechaty

## Project length

Standard length (3 months)

## PROJECT PROPOSAL GSOD 2021

User Doc is the most important integral part of any software be it paid or open source. The user docs site is designed to assist end-users to use a product or service. The good user docs site is very important because it provides an avenue for users to learn how to use the software, its features, tips, tricks and also resolve common problems encountered when using the software. It also reduces support cost and is part of the corporate identity of the product. The good user docs site is a sign of healthiness of the product, the developer team. Without good user docs site, a user may not know how to do the above things effectively and efficiently.
My personal ideas to the product would be that I will definitely create a new tutorial page which gives a detailed and structured introduction to the software from installation to the end-user usage.
the below are my personal ideas on the project:

- Markdown: - So that technical writers don't have to worry about installation. They can write simply in the .md file. Anyone can click on edit button shown on the website (new feature) and contribute making it better. This will engage users to add new content or edit content to improve it.
- Documentation Search: - User should be having a search box so that they can easily and quickly find out relevant contents.
- Comments Section: - User may have the option to comment and share their views on posts and tutorials. They can able to read other people views on project documentation.
- New Release Notes and Blogs: - The website should be updated with new blog posts and news about current development and roadmap.

Let me now, give a detailed description of the new website look

## Wechaty User Guide Page

- Installation guide.
- Quick start with Wechaty.
- Wechay Developer Guide Page:-
  Development Guide, Contribution guide, Git setup, Code of Conduct, Documentation setup
- About Wechaty Page:-
  List of all the contributors should be present in team sections Quick links and release notes, latest blogs will be added to engage the user to make them read about the latest Wechaty versions or so. Also read the uses of Wechaty
- Blogs, Release Notes and Tutorials Page:-
  The tutorial page will contain some popular talks and articles that clarify Wechay concepts. Contributors can add their tutorial links in the tutorial page.

## WHY AM I THE RIGHT PERSON FOR THIS PROJECT

I am the right person for the project because

- I have past experience of using Github and also been contributing to open source for more than an year now
- I have been top open-source contributor in many open-source contributions like Hackincodes and Girlscript Summer of Code.
- I have also been technical writer in GlobalCert Singapore and Girlscript summer of code for 2 months each
- I am also Medium blog writer and also at LoginRadius(My blockchain blog became rally famous)
- I am really interested in this project as it creates value for people and new bees who want to use Wechaty and also for the dedication with which I work in my current company my leaders are really happy and appreciate the dedicated work I do.
- I am also interested in working for Introduction ,Landing Page or References Page with a team member

## Links to some of my Technical Writing works

- <https://www.loginradius.com/blog/async/Blockchain-the-new-technology-of-security-trust/>
- <https://drive.google.com/drive/folders/1_g8lDR-Q0RXHTSj0vc3d4vvTXkrX2Ei0>
```

**Pull request closed without merging**